### PR TITLE
#8752 Fix Kpi Should not be allowed to be created before current date

### LINKED ISSUE #8752: Kpi Should not be allowed to be created before current date
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
UI

**Describe the bug**
Currently we can create the Kpi with a start date before the current date

**To Reproduce**

Screenshots or steps to reproduce

**Expected behavior**
A clear and concise description of what you expected to happeScreenshot 2022-11-15 at 1 14 38 PM.

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

UI should use a date picker instead of simple input for dates.

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Fixes #8752 
### Describe your changes :

I worked on issue #8752 
#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx [hunk 1]
```diff
@@ -20,11 +21,13 @@ import {
   last,
   toNumber,
 } from 'lodash';
+import moment from 'moment';
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx:24]: aren't we considering removing the moment?

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx:24]: Antd has a dependency of moment, so we have to use moment for antd date picker.

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx:24]: @chirag-madlani Any suggestion over here?

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx:24]: No @Sachin-chaurasiya that's not correct. Let's connect to resolve this

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx:24]: Sure @chirag-madlani

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx:24]: We have tried different ways to eliminate the moment but have had no luck so we will take care of eliminating the moment later.

**Comment by Sachin-chaurasiya** [comment]
@chirag-madlani @ShaileshParmar11 Comment has been addressed.

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**